Update to koa2

tiagojsag opened the pull request.

dependabot[bot] committed:
Bump websocket-extensions from 0.1.3 to 0.1.4

Bumps [websocket-extensions](https://github.com/faye/websocket-extensions-node) from 0.1.3 to 0.1.4.
- [Release notes](https://github.com/faye/websocket-extensions-node/releases)
- [Changelog](https://github.com/faye/websocket-extensions-node/blob/master/CHANGELOG.md)
- [Commits](https://github.com/faye/websocket-extensions-node/compare/0.1.3...0.1.4)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump websocket-extensions from 0.1.3 to 0.1.4

Bumps [websocket-extensions](https://github.com/faye/websocket-extensions-node) from 0.1.3 to 0.1.4.
- [Release notes](https://github.com/faye/websocket-extensions-node/releases)
- [Changelog](https://github.com/faye/websocket-extensions-node/blob/master/CHANGELOG.md)
- [Commits](https://github.com/faye/websocket-extensions-node/compare/0.1.3...0.1.4)

Signed-off-by: dependabot[bot] <support@github.com>

jterry64 committed:
Update production vars

jterry64 committed:
Update WDPA var

tiagojsag committed:
Update koa to v2 and associated dependencies; update to k8s 1.16

tiagojsag committed:
Update travis badge

tiagojsag committed:
Fix typo in jenkinsfile

ikas reviewed (approved):
Can't find any major issues, LGTM :+1:

dependabot[bot] committed:
Bump ini from 1.3.5 to 1.3.8

Bumps [ini](https://github.com/isaacs/ini) from 1.3.5 to 1.3.8.
- [Release notes](https://github.com/isaacs/ini/releases)
- [Commits](https://github.com/isaacs/ini/compare/v1.3.5...v1.3.8)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump ini from 1.3.5 to 1.3.8

Bumps [ini](https://github.com/isaacs/ini) from 1.3.5 to 1.3.8.
- [Release notes](https://github.com/isaacs/ini/releases)
- [Commits](https://github.com/isaacs/ini/compare/v1.3.5...v1.3.8)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot-preview[bot] committed:
[Security] Bump axios from 0.19.2 to 0.21.1

Bumps [axios](https://github.com/axios/axios) from 0.19.2 to 0.21.1. **This update includes a security fix.**
- [Release notes](https://github.com/axios/axios/releases)
- [Changelog](https://github.com/axios/axios/blob/v0.21.1/CHANGELOG.md)
- [Commits](https://github.com/axios/axios/compare/v0.19.2...v0.21.1)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
[Security] Bump axios from 0.19.2 to 0.21.1

Bumps [axios](https://github.com/axios/axios) from 0.19.2 to 0.21.1. **This update includes a security fix.**
- [Release notes](https://github.com/axios/axios/releases)
- [Changelog](https://github.com/axios/axios/blob/v0.21.1/CHANGELOG.md)
- [Commits](https://github.com/axios/axios/compare/v0.19.2...v0.21.1)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

tiagojsag committed:
Merge branch 'production' into dev

dependabot[bot] committed:
Bump lodash from 4.17.15 to 4.17.20

Bumps [lodash](https://github.com/lodash/lodash) from 4.17.15 to 4.17.20.
- [Release notes](https://github.com/lodash/lodash/releases)
- [Commits](https://github.com/lodash/lodash/compare/4.17.15...4.17.20)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump lodash from 4.17.15 to 4.17.20

Bumps [lodash](https://github.com/lodash/lodash) from 4.17.15 to 4.17.20.
- [Release notes](https://github.com/lodash/lodash/releases)
- [Commits](https://github.com/lodash/lodash/compare/4.17.15...4.17.20)

Signed-off-by: dependabot[bot] <support@github.com>

tiagojsag committed:
Replace CT integration library

tiagojsag committed:
Replace CT integration library

jterry64 committed:
Fix adm2 queries

jterry64 committed:
Fix adm2 queries

tiagojsag committed:
update dockerhub repo

tiagojsag committed:
Update RW API integration lib

tiagojsag committed:
Merge branch 'dev' into staging

# Conflicts:
#	package.json
#	yarn.lock
